src/AWS.Deploy.Orchestration/DeploymentSettingsHandler.cs

```diff
@@ -58,9 +58,13 @@ public DeploymentSettingsHandler(IFileManager fileManager, IDirectoryManager dir
                 var userDeploymentSettings = JsonConvert.DeserializeObject<DeploymentSettings>(contents);
                 return userDeploymentSettings;
             }
+            catch (IOException ex)
+            {
+                throw new InvalidDeploymentSettingsException(DeployToolErrorCode.DeploymentConfigurationNotFound, $"An error occurred while trying to read the deployment settings file located at {filePath}. Make sure the path you provided exists on your file system, and is readable.", ex);
```

Comment:
Before the `try` block can you call `_fileManager.Exists` and if that returns back false you can give a specific error message that the file does not exist.

Reply:
Done, please review the new changes.